news/5059.bugfix

```diff
@@ -0,0 +1 @@
+Remove last user of SafeConfigParser
```

Comment:
Something quick I noticed about this. I wouldn't mark it as a bugfix but rather trivial. (So no need to include a news file since it is marked trivial on the tracker.)

Reply:
I'd agree with you if I hadn't to write a test, but fine for me anyway :)

Reply:
What the test does is show others that the change you made actually works and doesn't break that code path! So it's not so much about whether or not a bug is present..

Reply:
Sure, i meant that the task is less trivial if you have to add a test from scratch, but i see you have a section in the documentation explaining what trivial means in this context :) Pushed with empty trivial news file.

Reply:
Oh, I see. You were referring to it being "trivial" rather than not being a "bugfix." Yes, you're right. "Trivial" doesn't mean "trivial" so much as "anything other than bugfix, feature, etc." :)